docs/maintaining-a-track.md

```diff
@@ -0,0 +1,79 @@
+# Maintaining an Exercism Language Track
+
+* [Philosophy](#philosophy)
+
+  * [Be Kind](#be-kind)
+  * [Be Responsive](#be-responsive)
+  * [Empower New Contributors](#empower-new-contributors)
+  * [Recognize Great Contributions](#recognize-great-contributions)
+  * [Improve the Learning Experience](#improve-the-learning-experience)
+  * [Collaborate](#collaborate)
+
+* [Day-to-Day Tactics](#day-to-day-tactics)
+
+  * [Watch Key Repositories](#watch-key-repositories)
+  * [Perform Common Tasks](#perform-common-tasks)
+
+## Philosophy
+
+### Be Kind
+
+First and foremost, strive always to be kind, humble and show gratitude for the work being contributed.
+
+**_Kind_ is not the same thing as _nice_.** Being kind does not mean being a pushover.
+Doing the right thing is not always the thing that pleases the most people.
+
+### Be Responsive
+
+Even if you do not have an answer immediately, respond quickly. This lets the person on the other end know that they've been heard. If it takes a long time to figure out what to do, be honest about what's going on.
+
+### Empower New Contributors
+
+As a maintainer, your most important task is to make it as easy as possible for people to successfully contribute. The exact details will vary, but often this means things like:
+
+- documenting or scripting the necessary steps to get started with the codebase
+- helping people understand how to use the tools successfully (including git and GitHub)
+- writing detailed issues about work that needs to be done so that someone without knowledge of the project can easily find something to work on
+- improving the tooling to reduce friction
+- answering questions
+- reviewing pull requests
+
+### Recognize Great Contributions
```

Comment:
Why only recognize "great" work? I think any contribution that helps is great, it doesn't have to be a great contribution :) This is one of the things I feel is very important: give contributors the feeling that you really value their work, no matter how small the contribution is. As a consequence, a track maintainer shouldn't just merge a PR, but should also thank the user for his or her contribution.